src/hooks/useOwnedVenues.tsx

```diff
@@ -13,7 +13,7 @@ import { useRefiCollection } from "hooks/fire/useRefiCollection";
 
 export interface UseOwnedVenuesOptions {
   worldId?: string;
-  userId: UserId;
+  userId: UserId | string;
```

Comment:
How many places call this hook? Can they be made `as UserId` instead?
I write this because most of the time (even more with newer in progress) hooks do return the type as `UserId`, not `string`.

Reply:
casting it `as UserId` didn't work. I don't remember why, but there was some kind of `_brand` error or something like that.